src/main/java/hudson/plugins/brakeman/BrakemanPublisher.java

```diff
@@ -60,7 +47,7 @@ public BrakemanPublisher(final String healthy, final String unHealthy, final Str
 				final String failedTotalAll, final String failedTotalHigh, final String failedTotalNormal, final String failedTotalLow,
 				final String failedNewAll, final String failedNewHigh, final String failedNewNormal, final String failedNewLow,
 				final boolean canRunOnFailed, final boolean shouldDetectModules, final boolean canComputeNew, final String outputFile) {
-
+			// TODO use non-deprecated constructor
```

Comment:
Consider filing an bug for this on the jenkins JIRA and add the link to it here.  TODO's are easy to overlook.